### PR TITLE
spread: refresh Fedora versions

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

**Review by Conan-Kudo — APPROVED** [review]

**Review by AlanGriffiths — APPROVED** [review]
Looks sensible. But waiting on CI passing Fedora builds before merging

**Comment by Conan-Kudo** [comment]
We should drop F38 since it will be EOL next month.

**Pull request merged**
